src/app/coursegrab/controllers/update_notification_controller.py

```diff
@@ -0,0 +1,17 @@
+from . import *
+from app.coursegrab.utils.constants import NONE
+
+
+class UpdateNotificationController(AppDevController):
+    def get_path(self):
+        return "/users/update-notification/"
+
+    def get_methods(self):
+        return ["POST"]
+
+    @authorize_user
+    def content(self, **kwargs):
+        data = request.get_json()
+        user = kwargs.get("user")
+        notification = None if data.get("notification") == NONE else data.get("notification")
```

Comment:
I'm pretty sure you can replace this whole line with just `notification = data.get("notification")`

Reply:
`NONE` is a constant to make parsing easy when we get requests